src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const largeImg = document.querySelector('#largeImg');
+const listItem = document.querySelectorAll('#thumbs a');
+
+listItem.forEach((thumb) => {
+  thumb.addEventListener('click', () => {
+    event.preventDefault(); 
```

Comment:
Bug: `event` is not defined in the scope of this function. It seems like you want to use the event object of the click event. You should pass it as an argument to the callback function.